content/php/concepts/math-functions/terms/atan/atan.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'atan()'
+Description: 'Returns the angle whose tangent value is equal to the specified value.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Math Functions'
+Tags:
+  - 'Methods'
+  - 'Trigonometric'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The inverse tangent function, which is used to identify the angle whose tangent value is equal to the specified value, is referred to as the **arc tangent** function.
+
+The built-in math function **`atan()`** determines the arc tangent of a given value. It is the inverse function of the [**`tan()`**](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
+
+> This means that `tan(atan(number))` would be equal to `number`.
+
+## Syntax
+
+The syntax of the `atan()` function is as follows:
+
+```pseudo
+$result = atan(number);
+```
+
+The atan() function takes one parameter:
+
+- $number (required): The number for which you want to calculate the arc tangent.
+
+Where `number` is the number whose arc tangent the function returns.
+
+## Return Value
+
+The `atan()` method returns a floating point number which is the arc tangent value of the number passed as argument.
+
+The resulting numeric value is between **-pi/2** and **pi/2** radians.
+
+To convert radians to degrees, we can use the following:
+
+```
+degrees = radians * ( 180.0 / M_PI )
+```
+
+## Example
+
+```php
```

Comment:
Even with a simple example we like an introductory sentence summarizing what it is doing.

Reply:
Added an introductory sentence.